src/test/java/fi/metatavu/kuntaapi/test/utils/PtvDownload.java

```diff
@@ -65,6 +157,97 @@ private void printIds() throws IOException {
+      
+      for (int i = 0; i < servicesTypeChannels.size(); i++) {
+        StringBuilder typeIds = new StringBuilder();
+        
+        typeIds.append("      {");
+        
+        EnumMap<ServiceChannelType, List<String>> serviceTypeChannels = servicesTypeChannels.get(i);
+        List<String> idList = serviceTypeChannels.get(channelType);
+        if (idList != null) {
+          List<String> ids = serviceTypeChannels.get(channelType).stream()
+            .map(id -> String.format(" \"%s\"", id))
+            .collect(Collectors.toList());
+          typeIds.append(StringUtils.join(ids, ","));
+        }
+        
+        typeIds.append(" }");
+        
+        channelTypeIds.add(typeIds.toString());
+      }
+      
+      System.out.println(StringUtils.join(channelTypeIds, ",\n"));
+      System.out.println("    };");
+    }
+  }
+  
+  private EnumMap<ServiceChannelType, List<String>> createChannelTypeMap() {
+    EnumMap<ServiceChannelType, List<String>> result = new EnumMap<>(ServiceChannelType.class);
+    
+    Arrays.stream(ServiceChannelType.values()).forEach((type) -> {
+      result.put(type, new ArrayList<>());
+    });
+    
+    Arrays.stream(getServiceChannelsFolder().listFiles()).forEach((channelFile) -> {
+      try {
+        ServiceChannelType channelType = resolveChannelType(channelFile);
+        String id = FilenameUtils.getBaseName(channelFile.getName());
+        result.get(channelType).add(id);
+      } catch (IOException e) {
+        e.printStackTrace();
+      }
+    });
+    
+    return result;
+  }
+
+  private List<EnumMap<ServiceChannelType, List<String>>> createServiceChannelsTypeMap(File[] serviceFiles) throws JsonParseException, JsonMappingException, IOException {
+    List<EnumMap<ServiceChannelType, List<String>>> result = new ArrayList<>();
+    
+    for (int i = 0; i < serviceFiles.length; i++) {
+      File serviceFile = serviceFiles[i];
+      EnumMap<ServiceChannelType, List<String>> channels = new EnumMap<>(ServiceChannelType.class);
+      
+      V8VmOpenApiService service = new ObjectMapper().readValue(serviceFile, V8VmOpenApiService.class);
+      List<V8VmOpenApiServiceServiceChannel> serviceChannels = service.getServiceChannels();
+      for (V8VmOpenApiServiceServiceChannel serviceChannel : serviceChannels) {
+        File serviceChannelFile = new File(getServiceChannelsFolder(), String.format("%s.json", serviceChannel.getServiceChannel().getId()));
+        ServiceChannelType channelType = resolveChannelType(serviceChannelFile);
+        if (!channels.containsKey(channelType)) {
+          channels.put(channelType, new ArrayList<>());
+        }
+        
+        channels.get(channelType).add(serviceChannel.getServiceChannel().getId().toString());
+      }
+      
+      result.add(channels);
+    }
+
+    return result;
+  }
+
+  private ServiceChannelType resolveChannelType(File file) throws JsonParseException, JsonMappingException, IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)